scripts/ecc-heap.sh

```diff
@@ -54,32 +54,45 @@ cat << EOF >$CONFIG_H
 
 #define MBEDTLS_BIGNUM_C
 #define MBEDTLS_ECP_C
+#define MBEDTLS_ECP_NO_INTERNAL_RNG
 #define MBEDTLS_ASN1_PARSE_C
 #define MBEDTLS_ASN1_WRITE_C
 #define MBEDTLS_ECDSA_C
+#define MBEDTLS_SHA256_C // ECDSA benchmark needs it
+#define MBEDTLS_SHA224_C // SHA256 requires this for now
```

Comment:
This line has no effect in 2.28.

Reply:
Yeah, I wasn't sure if I should remove it because it's useless, or leave it for uniformity with development. Then I forgot about it, so it stayed.